### ISSUE TITLE
'hello world': Bb is not recognized

### ISSUE THREAD

**Issue by ruicsh** [Squareknot/marionette.state#19]
I can't seem to make it work by simply including the latest version of the dependencies (no module loader). Here's a [gist using your first example, Stateful View](https://gist.github.com/ruicosta042/87ce329f9505e380abfa)

error here: 
`// State model class is either passed in, on the class, or a standard Backbone model
this.modelClass = this.modelClass || Bb.Model;`

`Uncaught TypeError: Cannot read property 'Model' of undefined`

`Bb` is not recognized. What am I doing wrong?

**Comment by ianmstew** [comment]
@ruicosta042 Thanks for reporting this!  It looks like the alias `Bb` that we're using internally for Backbone is actually causing the bundle to look for `window.Bb`, which doesn't exist.  We're working on a fix asap, but you can get it working immediately by adding the following script tag after Backbone and before Marionette.State:

``` html
<script src="path/to/backbone"></script>
...
<script>
window.Bb = window.Backbone
</script>
...
<script src="path/to/marionette.state"></script>
```

**Comment by ruicsh** [comment]
@ianmstew thanks for this. This made it work on the simple example. But, I can't use it on our live Marionette application: we're using .noConflict() versions of Backbone and Marionette, not even using window.Backbone, but our own namespaced var (ex.: MyApp.framework.Marionette). Thanks anyway, we'll wait for more updates from you. cheers

**Comment by ianmstew** [comment]
Sorry for the delay, hit a job transition but I'm back in business.  Update coming shortly.

**Comment by ianmstew** [comment]
Fixed in `v0.4.1`, just published.
